app/src/main/java/org/groundplatform/android/ui/map/gms/GoogleMapsFragment.kt

```diff
@@ -281,21 +283,47 @@ class GoogleMapsFragment : SupportMapFragment(), MapFragment {
 
   override fun addTileOverlay(source: TileSource) =
     when (source) {
-      is LocalTileSource -> addLocalTileOverlay(source.localFilePath, source.clipBounds)
-      is RemoteMogTileSource -> addRemoteMogTileOverlay(source.remotePath)
+      is LocalTileSource ->
+        addLocalTileOverlay(source.localFilePath, source.clipBounds, source.maxZoom)
+      is RemoteMogTileSource ->
+        addRemoteMogTileOverlay(url = source.remotePath, zIndex = TILE_OVERLAY_Z)
```

Comment:
Can we inline `TILE_OVERLAY_Z` as `addRemoteMogTileOverlay` is only called from this caller.